### ISSUE TITLE
Stop overnight updating of UserID on the image server

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#1466]

**Comment by edickens** [comment]
The UserID folders on the image server are updated overnight with new UserIDs and changes in password.  All new transcribers should now start on the website image assignment system.  All regular transcribers are now on the website assignment system.  Stopping the overnight updating will not affect any transcribing that is in progress.
When overnight updating has stopped, I want to delete the unused UserIDs on the image server and check that any work I find is still being transcribed.  There is a lot outstanding.  These transcribers need to be moved to the server system.
EricD

**Comment by richpomfret** [comment]
@Captainkirkdawson Please take a look and see if this change won't result in anything 'falling over'.

**Comment by Captainkirkdawson** [comment]
I do not see how stopping the update helps the clean up. If transcribers had access they will still have access. I think the clean up of folders is independent of this stated requirement.

**Comment by edickens** [comment]
New transcribers do not use a folders on the image server any more.  Those few who do use getscan  need to be moved off that system.  Yes, I will need to do the cleanup.  I want to go through every one of the thousands of folders to check that there are no assigned images there.  The old system recording of assignments was manual and not checked for nearly 20 years.  So I need to look in each folder, and if empty, delete it.  But if the overnight processing continues, the next day that folder will be reinstated.
These folders are no longer needed and there are thousands of empty folders.
![capture](https://user-images.githubusercontent.com/8876165/40962744-4e94b1e8-689e-11e8-9442-785ef33f4e4e.JPG)

**Comment by edickens** [comment]
PS.  There are folders for those who registered but never transcribed.  And the overnight update adds but never deletes.
Deleting these folders has no effect on the server or on anyone else other than me.

**Comment by Captainkirkdawson** [comment]
Now clear why it is needed

**Comment by edickens** [comment]
Or why it is not needed.  Cheers  Eric

**Comment by Captainkirkdawson** [comment]
syncing has been stopped and creation of table no longer occurs

**Comment by edickens** [comment]
TY.  Will do a sample check in case the old system has something else hidden away.

**Comment by edickens** [comment]
Some UserID folders are still being replaced after I have deleted them.  I suspect that there is a getscan routine that is still being run overnight which is recreating these folders.  It is probably not part of the FR overnight routines, but a FR1 routine which Lemon will need to edit.  Deleting the transfer of information file may cause something to crash.  Can someone please look into this.

**Comment by Captainkirkdawson** [comment]
Thought something like that might happen. I could try creating an empty file instead and allow that to be synced. There are no update parts of FR1 running. But the original code that getscan is part of is still there

**Comment by edickens** [comment]
Yes please Kirk.   I think an empty file would work.  Something must be working overnight and not just when  someone runs getscan.  But not important to find out what it is, we just need to work around it.

**Comment by Captainkirkdawson** [comment]
@edickens I have created a file with just your credentials. Re-instituted the sync. Suggest you make some deletions and check tomorrow

**Comment by Captainkirkdawson** [comment]
That is what I have done but the sync is needed to get it over to the image
server

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by edickens** [comment]
The folders have reappeared.  Thoughts?  Eric

**Comment by edickens** [comment]
Can you see if the file with UserIDs and passwords only has my UserID on it?  Even the file size will do, because the original would have had 2,000 or more records.  It should be very small now.
I may have to go to Lemon if this does not work.

**Comment by edickens** [comment]
Can you see if the file with UserIDs and passwords only has my UserID on it?  Even the file size will do, because the original would have had 2,000 or more records.  It should be very small now.
I may have to go to Lemon if this does not work.

**Comment by Captainkirkdawson** [comment]
I see what happened. The change to stop creating the REGusers file had not been deployed into production hence there was a full recreation. Should be deployed latter today

**Comment by edickens** [comment]
Ty.  I'll delete again and check tomorrow.  E

**Comment by edickens** [comment]
Has worked.  This can be closed.

**Comment by edickens** [comment]
Doreen and I can no longer log in to Getscan.  Nobody else has complained.  Not a problem and I suggest that we say it has now finally died.  Cheers Eric

**Comment by Captainkirkdawson** [comment]
Buts that,s what you asked for!

EricD can still log in

**Comment by SteveBiggs** [comment]
My understanding was that the overnight updating which ADDED new userids would be stopped. I didn't expect it to remove all current userids. It may not be a problem, but there are some transcribers working on batches assigned many months ago though getscan who I was planning to move to the new system when they're ready for their next batch. If they've already downloaded all the files then that's fine, if not, I will transfer them to access the remaining images via the new system.

**Comment by Captainkirkdawson** [comment]
But Eric stated 
So I need to look in each folder, and if empty, delete it. But if the overnight processing continues, the next day that folder will be reinstated.
These folders are no longer needed and there are thousands of empty folders.
**That as nothing to do with adding new userid but deleting old ones.**
If you look at the history of this story you will see the proposal to create a one record file that was accepted and then implemented

**Comment by edickens** [comment]
They have not been removed.  The information is all still there.  It is just that for some reason the passwords don;t work.  No idea why because we have not done anything to change them.  It is the getscan overnight processing that seems to have done something.  And I've tried with no password.
This is the start of the list of remaining UserIDs from which I have removed any with no images.
![capture](https://user-images.githubusercontent.com/8876165/41295611-e2fac192-6e52-11e8-8595-390008ae2062.JPG)
I check every one and have found images in UserIDS which are not in the pool and so not referenced through FR, a situation that I am correcting.

**Comment by Captainkirkdawson** [comment]
This whole exchange is totally incredible.
Your statement
"It is just that for some reason the passwords don;t work. No idea why because we have not done anything to change them. It is the getscan overnight processing that seems to have done something. "

**_You explicitly agreed that we would eliminate them_**

**Passwords don't work because we eliminated them as you asked me to do because if we did not getscan would recreate the folder each time you deleted it**

**Comment by edickens** [comment]
But the getscan overnight processing leaves everything even when a UserID and its password is deleted.  There is a load of rubbish there.  So if you created a file with no UserIDs, then those on the image server still stay, and they are all still there.  It is just that it seems to have changed, and not lost, the passwords.  Everything else is fine.  This is not a problem as it will force transcribers onto FR.  I can still access everything with BitKinex.

**Comment by Captainkirkdawson** [comment]
I give up!

**Comment by edickens** [comment]
What you have done is great.  Nothing more for you to do, with this!  I am getting it sorted and have found missing images.
